vsintegration/src/FSharp.Editor/CodeFixes/RemoveUnusedOpens.fs

```diff
@@ -30,14 +31,20 @@ type internal RemoveUnusedOpensCodeFixProvider [<ImportingConstructor>] () =
             cancellableTask {
                 let! sourceText = context.GetSourceTextAsync()
 
-                let span =
-                    sourceText.Lines.GetLineFromPosition(context.Span.Start).SpanIncludingLineBreak
+                let! unusedOpensOpt = UnusedOpensDiagnosticAnalyzer.GetUnusedOpenRanges context.Document
 
                 return
-                    ValueSome
+                    unusedOpensOpt
```

Comment:
Probably should make `GetUnusedOpenRanges` return `Option` instead of converting, if possible.

Reply:
There you go. Needed some collateral changes but not too far, should be fine.